sig-contributor-experience/charter.md

```diff
@@ -0,0 +1,59 @@
+# Contributor Experience Special Interest Group Charter
+*checking in for comments and will add TODOs at bottom along with feedback that comes in*
+
+The Contributor Experience Special Interest Group is dedicated to ensuring that Kubernetes contributors have a consistent workflow and great experience within the ecosystem. Developing and sustaining a healthy community of contributors is critical to scaling the project and growing the folks who build it. We champion the needs of these contributors by building programs and processes that address any gaps, learning and development, reward and recognize positive actions, and facilitiate project wide communications. 
```

Comment:
Introduce acronym SIG at this point as it is the popular term and the doc later uses it.. e.g.  "The Contributor Experience Special Interest Group (SIG) is dedicated to..."